docs/install.rst

```diff
@@ -11,6 +11,12 @@ First, obtain `Python 3.6`_ and virtualenv_ if you do not already have them.
 Using a virtual environment is strongly recommended,
 since it will help you to avoid clutter in your system-wide libraries.
 
+.. note::
```

Comment:
This should probably be a `warning` instead.

Reply:
I'd really like to keep these edge cases out of the default flow for the install guide, otherwise it just gets messy. We should add them to a troubleshooting section at the bottom of the guide, with the error that they fix.

Reply:
The thing is that it's not an edge case.

All users installing Python 3.6 with an up to date system will hit this problem. The issue is fixed on Django 2.x but we are using 1.11.x. So, all new contributors will hit it until we upgrade our django version.

Reply:
OK, thanks sense then.

Reply:
OK, thanks sense then.